### PR TITLE
Update JSON-RPC.md with the new RPC 'mempool_deleteAllPendingTransactions()'

### PR DESCRIPTION
**Pull request by junha1**
Regarding https://github.com/CodeChain-io/codechain/pull/1941

### CODE REVIEW

File: spec/JSON-RPC.md [hunk 1]
```diff
@@ -1727,6 +1727,29 @@ Errors: `Invalid Params`
 
 [Back to **List of methods**](#list-of-methods)
 
+## mempool_deleteAllPendingTransactions
+Delete all pending transactions in both current and future queue.
```
Comment (majecty) [inline spec/JSON-RPC.md]: We are using a third-person singular verb in the description. We are assuming that the method name is the subject.

Comment (junha1) [reply spec/JSON-RPC.md]: Ok

**Comment by majecty** [comment]
Use more specific commit title and PR title. Please summarize what you added in the spec file.

**Comment by junha1** [comment]
@majecty Should I directly mention the name of the new RPC in the PR title & commit message?

**Comment by majecty** [comment]
Yes, please mention the name of the new RPC.

**Comment by majecty** [comment]
@junha1 Please make the commit title the same as the PR title.

**Pull request merged**